uws/UWS/base.py

```diff
@@ -41,6 +41,25 @@ def get_job(self, id):
 
         return result
 
+    def get_phase(self, id):
+        try:
+            response = self.connection.get(id + '/phase')
+        except Exception as e:
+            raise UWSError(str(e))
+
+        raw = response.read()
+        # response is only the phase name, there is exactly one phase per job, 
+        # so just use this here.
+        try:
+            result = raw
```

Comment:
I think you don't need the try catch here. Since raw will just contain the phase name, you can just return this. So you can get rid of the whole try catch block and just return raw.